eclipse-collections-api/src/main/java/org/eclipse/collections/api/bimap/ImmutableBiMap.java

```diff
@@ -134,6 +138,36 @@
         return this.groupByUniqueKey(function, target).toImmutable();
     }
 
+    @Override
```

Comment:
Hi @motlin, what do you think about adding since tags to these methods so we can easily see when they were added?

Reply:
Done